tests/integration/test_s3_table_function_with_http_proxy/configs/config.d/proxy_list.xml

```diff
@@ -2,7 +2,6 @@
     <proxy>
         <http>
             <uri>http://proxy1</uri>
-            <uri>http://proxy2</uri>
```

Comment:
I do not understand why did you remove all second records?

Reply:
I still do not understand. May be I miss something it was a long thread.

Reply:
I don't quite remember either. I am sure it was smething related to the test refactoring in this PR that was going to bit a bit harder if we were to keep multiple entries, and since it is already tested in [unit tests](https://github.com/ClickHouse/ClickHouse/blob/72372425766cec1fd713be71cf75b4b426f69244/src/Common/tests/gtest_proxy_list_configuration_resolver.cpp#L15), I thought it would be easier to just remove it.

I'll try to introduce it back.

Reply:
Done